scripts/warm-dotio-cache.ts

```diff
@@ -107,7 +107,7 @@ async function getTutorialUrlsToCache(
 
 		const tutorialUrls = (
 			await Promise.all(
-				BETA_PRODUCTS.map((product: ProductOption) =>
+				productSlugs.map((product: ProductOption) =>
 					getTutorialUrlsToCache(product)
```

Comment:
💭 just realized we need to account for the hcp -- cloud map here. I can make a follow up ticket, not necessary on this PR.

Reply:
We could also adjust this now to just fetch _everything_ instead of making one req per product.

Reply:
https://app.asana.com/0/1202702246158644/1203081935898618